woocommerce/woocommerce-gutenberg-products-block
# Add Cart Line Items Block to Cart i2

Adds Cart Line Items block (Couldn't use Cart Item Block because it's already used).

Part of #4709, based on #4780

## Testing Steps
- Insert Cart i2.
- Make sure you can see items.
- Make sure it's using preview data.
